tests/test_single_topology_combined.py

```diff
@@ -13,32 +13,42 @@
 from timemachine.testsystems.relative import get_hif2a_ligand_pair_single_topology
 
 
-def test_combined_parameters_solvent():
+@pytest.fixture(scope="module")
+def hif2a_ligand_pair_single_topology():
+    mol_a, mol_b, core = get_hif2a_ligand_pair_single_topology()
     forcefield = Forcefield.load_from_file(DEFAULT_FF)
-    host_sys_omm, _, _, _ = builders.build_water_system(3.0, forcefield.water_ff)
-    _test_combined_parameters_impl_bonded(host_sys_omm)
-    _test_combined_parameters_impl_nonbonded(host_sys_omm)
+    return SingleTopology(mol_a, mol_b, core, forcefield)
+
 
+def convert_omm_system(omm_system):
```

Comment:
nit[code location] -- looks potentially convenient outside of this test.

Should this instead live in `timemachine.fe.system`, near `convert_bps_into_system`?

Reply:
Good call. Moved in 313135f and updated `rbfe.py` to use this in 64f18f.